CMakeLists.txt

```diff
@@ -166,6 +167,7 @@ mark_as_advanced(QUDA_DEVICE_PACK)
 mark_as_advanced(QUDA_SSTEP)
 mark_as_advanced(QUDA_EIGEN)
 mark_as_advanced(QUDA_BLOCKSOVER)
+mark_as_advanced(QUDA_DEFLATEDSOVER)
```

Comment:
Typo there, should be `QUDA_BLOCKSOLVER` and `QUDA_DEFLATEDSOLVER`.  Either way, I'm sure these methods count as advanced any more, so I would suggest deleting lines 169 and 170.